### ISSUE TITLE
Start Errors

### ISSUE THREAD

**Issue by daslicht** [capaj/jspm-mithril#2]
Hi,
I did the following:

```
git clone git@github.com:capaj/jspm-react.git
cd jspm-react
npm install
npm start
```

I get :

```
Failed to load resource: the server responded with a status of 404 (Not Found)
http://localhost:9080/capaj/systemjs-hot-reloader Failed to load resource: the server responded with a status of 404 (Not Found)
system.js:4 Uncaught (in promise) Error: XHR error (404 Not Found) loading http://localhost:9080/capaj/systemjs-hot-reloader(…)
```

**Comment by capaj** [comment]
@daslicht should be fixed in 61be9cb6, let me know if you still have any problems

**Comment by daslicht** [comment]
Hm funny , how do I get this giturl : ithub.com:capaj/jspm-react.git
Seams to work now!
